<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{ 
  background-color:rgba(50,50,50,255); 
  color:rgb(222,222,222); 
   
} 
QPushButton{ 
  color:rgb(238,138,65); 
  border:none; 
  border-width:5px; 
  border-image: url(:/rs/btn_black_normal.png)  5; 
  min-width:40px; 
  min-height:40px; 
} 
QPushButton:hover{ 
  border:none; 
  border-width:5px; 
  border-image: url(:/rs/btn_black_hover.png)  5; 
} 
QPushButton:pressed{ 
  border:none; 
  border-width:5px; 
  border-image: url(:/rs/btn_black_pushed.png)  5; 
} 
 
QPushButton:disabled{ 
  border:none; 
  border-width:5px; 
  border-image: url(:/rs/btn_black_disable.png)  5; 
} 
 
QLineEdit{ 
  border-width:4px; 
  border-image: url(:/rs/edit_normal.png) 4 ; 
  min-height:32px; 
} 
QLineEdit:hover{ 
  border-image: url(:/rs/edit_hover.png) 4 ; 
} 
QLineEdit:focus  { 
  border-image: url(:/rs/edit_hover.png) 4 ; 
} 
 
QComboBox { 
   border-width:4px; 
  border-image: url(:/rs/edit_normal.png) 4; 
  border-image: url(:/rs/edit_hover.png) 4; 
  margin-right:52px; 
  min-height:36px; 
} 
QComboBox:hover{ 
  border-image: url(:/rs/edit_hover.png) 4 ; 
} 
QComboBox:focus{ 
  border-image: url(:/rs/edit_hover.png) 4 ; 
} 
QComboBox::drop-down { 
      subcontrol-origin: margin; 
      subcontrol-position: top right; 
      width:52px; 
     border:none; 
  } 
QComboBox::down-arrow{ 
  image: url(:/rs/downn_cmb_btn_normal.png); 
  } 
QComboBox::down-arrow:hover{ 
  image: url(:/rs/down_cmb_btn_hover.png); 
  } 
QComboBox::down-arrow:pressed{ 
  image: url(:/rs/down_cmb_btn_pushed.png); 
  } 
 
 
QRadioButton::indicator::unchecked { 
    image: url(:/rs/radio_unselect.png); 
} 
QRadioButton::indicator::unchecked:hover{ 
    image:url(:/rs/radio_unselect_hover.png); 
} 
 
QRadioButton::indicator::checked { 
     image: url(:/rs/radio_selected.png); 
} 
QRadioButton::indicator::checked:hover { 
     image: url(:/rs/radio_selected_hover.png); 
} 
 
QPlainTextEdit{ 
    border-width:4px; 
  border-image: url(:/rs/edit_normal.png) 4 ; 
  min-height:64px; 
} 
 
QTabWidget::tab-bar { 
      left: 5px; /* move to the right by 5px */ 
  } 
 
  /* Style the tab using the tab sub-control. Note that 
      it reads QTabBar _not_ QTabWidget */ 
  QTabBar::tab { 
    height:32px; 
  } 
QTabBar::tab:selected { 
      color:rgb(235,135,65); 
  } 
QTabBar::tab:disabled { 
      color:rgb(125,125,125); 
  } 
QScrollBar:vertical { 
       border: none; 
    border-top-width:12px; 
    border-bottom-width:12px; 
    border-image: url(:/rs/v_scroll_scallbar.png) 12 0; 
       width: 24px; 
       margin:16px 0; 
} 
QScrollBar::handle:vertical { 
       border-width:5 2; 
    min-height:32px; 
    border-image: url(:/rs/v_scroll_handle.png) 5 2; 
    image:url(:/rs/v_scroll_handle_image.png); 
} 
QScrollBar::handle:vertical:hover{ 
    border-image: url(:/rs/v_scroll_handle_hover.png); 
 } 
QScrollBar::handle:vertical:pressed{ 
    border-image:url(:/rs/v_scroll_handle_pressed.png); 
 } 
 
QScrollBar::add-line:vertical { 
       border:none; 
    margin-top:4px; 
       height: 12px; 
    width:16px; 
       subcontrol-position: bottom; 
       subcontrol-origin: margin; 
} 
 
QScrollBar::sub-line:vertical { 
       border:none; 
    margin-bottom:4px; 
       height: 12px; 
    width:16px; 
       subcontrol-position: top; 
       subcontrol-origin: margin; 
} 
 
QScrollBar::up-arrow:vertical{ 
  border:none; 
   width:16px; 
  height:12px; 
  image:url(:/rs/v_scroll_btn_up.png); 
} 
QScrollBar::up-arrow:vertical:hover{ 
  image:url(:/rs/v_scroll_btn_up_hover.png); 
} 
QScrollBar::up-arrow:vertical:pressed{ 
  image:url(:/rs/v_scroll_btn_up_pressed.png); 
} 
 
QScrollBar::down-arrow:vertical{ 
  border:none; 
   width:16px; 
  height:12px; 
  image:url(:/rs/v_scroll_btn_down.png); 
} 
QScrollBar::down-arrow:vertical:hover{ 
  image:url(:/rs/v_scroll_btn_down_hover.png); 
} 
QScrollBar::down-arrow:vertical:pressed{ 
  image:url(:/rs/v_scroll_btn_down_pressed.png); 
} 
 
 
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical { 
   background: none; 
} 
 
QScrollBar:horizontal { 
    border: none; 
  border-left-width:12px; 
  border-right-width:12px; 
  border-image: url(:/rs/h_scroll_scallbar.png)  0 12; 
   height: 24px; 
   margin:0 16px; 
} 
 
QScrollBar::handle:horizontal { 
   border-width:2 5; 
  min-width:32px; 
  border-image: url(:/rs/h_scroll_handle.png) 2 5; 
  image: url(:/rs/h_scroll_handle_image.png); 
} 
 
QScrollBar::handle:horizontal:hover{ 
  border-image: url(:/rs/h_scroll_handle_hover.png); 
} 
 
QScrollBar::handle:horizontal:pressed{ 
  border-image:url(:/rs/h_scroll_handle_pressed.png); 
} 
 
 
QScrollBar::add-line:horizontal { 
   border:none; 
  margin-right:4px; 
   height: 12px; 
  width:16px; 
   subcontrol-position: right; 
   subcontrol-origin: margin; 
} 
 
QScrollBar::sub-line:horizontal { 
   border:none; 
  margin-left:4px; 
   height: 12px; 
  width:16px; 
   subcontrol-position: left; 
   subcontrol-origin: margin; 
} 
 
 
 
 
QScrollBar::left-arrow:horizontal{ 
  border:none; 
   width:12px; 
  height:16px; 
  image:url(:/rs/h_scroll_btn_left.png); 
} 
QScrollBar::left-arrow:horizontal:hover{ 
  image:url(:/rs/h_scroll_btn_left_hover.png); 
} 
QScrollBar::left-arrow:horizontal:pressed{ 
  image:url(:/rs/h_scroll_btn_left_pressed.png); 
} 
 
QScrollBar::right-arrow:horizontal{ 
  border:none; 
   width:12px; 
  height:16px; 
  image:url(:/rs/h_scroll_btn_right.png); 
} 
QScrollBar::right-arrow:horizontal:hover{ 
  image:url(:/rs/h_scroll_btn_right_hover.png); 
} 
QScrollBar::right-arrow:horizontal:pressed{ 
  image:url(:/rs/h_scroll_btn_right_pressed.png); 
} 
 
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal { 
     background: none; 
} </string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>89</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabHuobi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>511</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>交易对</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>交易量(btc)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>深度延时</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>历史延时</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>交易所</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbMarket"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tabBids">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>400</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tabAsks">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>231</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>83</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabTradeHistory">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>681</width>
     <height>331</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>direction</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ts</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="edtTest2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>113</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtTest3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>113</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>246</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>平均历史延时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtAvgHistoryDelay"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>246</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="xxxxxxxxx">
      <property name="text">
       <string>平均深度延时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtAvgDepthDelay"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
